ig/input/fsh/nbs-pid-segment.fsh

```diff
@@ -3,23 +3,29 @@ Id: segment-pid-logical-model
 Title: "PID Segment"
 Description: "The PID Segment"
 * segmentType 1..1 SU string "PID"
-* setId 1..1 SU string "PID-1 a sequence number"
-* patientId 0..1 SU string "a patient id (retained for backward compatibility, but should be blank"
-* patientIdList 1..1 SU string "PID-3 a list of patient identifiers"
-* alternatePatientId 0..1 SU string "alternate patient id, should be blank"
-* patientName 1..1 SU string "PID-5 patient name"
-* mothersMaidenName 0..1 SU string "PID-6 mothers maiden name"
-* birthDateTime 1..1 SU string "PID-7 birth date and time"
+* setId 1..1 SU positiveInt "PID-1 a sequence number"
```

Comment:
Is the first letter in the sentence meant to be capitalized (i.e. `A sequence number`)? I'm not familiar with the format, so I'm not sure

Reply:
I think that's a formatting choice for us, but we should probably be uniform in our choice - I'll clean this up on the branch and make a commit.